src/status_im/protocol/web3/delivery.cljs

```diff
@@ -85,23 +87,33 @@
 (s/def :delivery/pending-message
   (s/keys :req-un [:message/sig :message/to :shh/payload :payload/ack? ::id
                    :message/requires-ack? :message/topic ::attempts ::was-sent?]
-          :opt-un [:message/pubKey :message/symKeyID]))
+          :opt-un [:message/pub-key :message/sym-key-password]))
 
-(defn add-prepared-pending-message!
-  [web3 {:keys [message-id to sym-key-id pub-key] :as pending-message}]
-  {:pre [(valid? :delivery/pending-message pending-message)]}
-  (debug :add-prepared-pending-message!)
-  (let [message          (assoc
-                           (select-keys pending-message [:sig :topic :payload])
-                           :symKeyID sym-key-id
-                           :pubKey pub-key)
-        pending-message' (assoc pending-message :message message
+(defn- do-add-pending-message!
+  [web3 {:keys [message-id to pub-key sym-key-id] :as pending-message}]
+  (let [message          (select-keys pending-message [:sig :topic :payload])
+        message'         (if sym-key-id
+                           (assoc message :symKeyId sym-key-id)
```

Comment:
Do we keep the old key for backward compatibility?

Reply:
I don't understand the question? What is old key here?

Reply:
Nevermind I read too fast.
Any reason we cannot use camelcase for `symKeyId ` too?

Reply:
The only reason why we are using camel case here is that web3 requires this. Why should we use it for other symbols/keywords?

Reply:
I meant clojure case actually ;)
And `sym-key-password` is fine? What caught my eye is the case difference.

Reply:
at some point we will need to use camelcase for `symKeyId` and `pubKey`, and as we are not going to use both after adding them to `message` it's fine to use camel case here. Later we will transform `message` with `clj->js`.